src/main/java/org/fundacionjala/coding/manuel/CesarVigenereCrypto.java

```diff
@@ -0,0 +1,133 @@
+package org.fundacionjala.coding.manuel;
+import java.util.StringJoiner;
+
+/**
+ * CesarVigenereCrypto class.
+ */
+public final class CesarVigenereCrypto {
```

Comment:
Split in two classes Cesar and Vigenere and apply abstraction for encode and decode